torch/csrc/jit/codegen/cuda/scheduler/pointwise.cpp

```diff
@@ -391,6 +391,12 @@ class DomainMap {
     return nullptr;
   }
 
+  static bool hasReferenceTensorView(Fusion* fusion) {
+    FusionGuard fg(fusion);
+    DomainMap domain_map(fusion);
+    return domain_map.findReferenceTensorView() != nullptr;
+  }
```

Comment:
Is `areAllMapped` correct? A little worried about line 420/426 below, we could skip mapping any axes that map to an output broadcast, but why would we skip a broadcast on an input if it is resolved? I think it should be:
```.cpp
    for (auto in_id : input_tv->getMaybeRFactorDomain()) {
      if (!ca_index_map_.getConcreteMappedID(in_id)->isBroadcast() && !in_id->isReduction()) {
        in_concrete_ids.insert(ca_index_map_.getConcreteMappedID(in_id));
      }
    }
```

Reply:
I think the view portion looks fine, but please double check that as well as this.

Reply:
It makes sense to check any input broadcast axes if they are mapped to a concrete iterDomain (e.g., resolved broadcast).

Reply:
Updated. Thanks!